src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml

```diff
@@ -311,7 +314,17 @@
         DismissClick="_loadingStatusBar_DismissClick"
         Style="{StaticResource FadeAnimationStyle}"
         Visibility="Hidden"/>
+
+      <!--
+        Special TextBlock to report loading status to assistive technologies (narrator).
+        Code behind changes in Text property trigger a narrator event
```

Comment:
Either omit this sentence or reword. You're trying to say that any Text change, even from codebehind will trigger this LiveTextBlock ?

Reply:
That's correct. Any changes to the Text raises a event to the narrator.

Reply:
I understand that, but this comment is still confusing to me.
```suggestion
        Code behind changes to the Text property will trigger a narrator event.
```

Reply:
Applied your suggestion.